recipes/simsopt/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "simsopt" %}
+{% set version = "0.3.03" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hiddenSymmetries/simpsopt/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5201fcd8f53857791c2d8e18fe608845e4d18b6c25eac96625d47700a52df87f
+
+build:
+  number: 0
+  skip: "True  # [win, py<37]"
```

Comment:
```suggestion
  skip: True  # [win or py<37]
```

Reply:
Thank you! I was struggling with that for a while since the documentation was scarce on that point.